bin/morgue.js

```diff
@@ -4676,11 +4676,8 @@ function argvQuery(argv) {
         errx('Filter must be of form <column>,<operation>[,<value>].');
       }
 
-      if (r[0] == "_tx" && r.length == 3 && typeof r[2] === "string") {
-        var rr = r[2].split("x");
-        if (rr.length === 2) {
-          r = [r[0], r[1], parseInt(rr[1], 16)];
-        }
+      if (r[0] == "_tx" && r.length == 3) {
+        r = [r[0], r[1], parseInt(r[2])];
```

Comment:
Do we want to leave `r[2]` alone if `parseInt` fails (returns a NaN)? This will hopefully lead to clearer error messages on malformed inputs.

Reply:
Makes sense.  I'll add an `|| r[2]` to allow for this